profiles/polkit-default-privs.restrictive

```diff
@@ -494,11 +494,11 @@ org.libvirt.api.network-port.read			auth_self:yes:yes
 org.libvirt.api.domain.checkpoint                       no 
 
 # MATE settings-daemon (bnc#831404)
-org.mate.settingsdaemon.datetimemechanism.settimezone		auth_admin_keep
-org.mate.settingsdaemon.datetimemechanism.settime		auth_admin_keep
-org.mate.settingsdaemon.datetimemechanism.configurehwclock	auth_admin_keep
-org.mate.randr.install-system-wide                              auth_admin_keep
-org.mate.power.backlight-helper					auth_admin_keep:auth_admin_keep:yes
+org.mate.settingsdaemon.datetimemechanism.settimezone		no:no:auth_admin_keep
+org.mate.settingsdaemon.datetimemechanism.settime		no:no:auth_admin_keep
+org.mate.settingsdaemon.datetimemechanism.configurehwclock	no:no:auth_admin_keep
+org.mate.randr.install-system-wide				no:no:auth_admin_keep
```

Comment:
this doesn't match the commit message, was this included by accident?

Reply:
Hmm I can't see which commit exactly you're refering to. Do you mean commit 475be363020d829278962d4609819c6cc58376ef? There I changed more stuff in the restrictive profile than in the others, but it is intended.

Reply:
yes that was the commit. Okay, I was just a bit unsure since you described pretty much every other change in the commit message.